scripts/smart-transcript-sync.py

```
@@ -0,0 +1,822 @@
+            return json.loads(CLARIFICATION_QUEUE_FILE.read_text())
+        except json.JSONDecodeError:
+            return []
+    return []
+
+
+def save_clarification_queue(queue: list):
+    """Save pending clarification queue"""
+    CLARIFICATION_QUEUE_FILE.write_text(json.dumps(queue, indent=2))
+
+
+def queue_for_clarification(memory: dict, source_session: str = None, 
```

Ruff (0.15.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)